packages/components/src/components/dialog/dialog.stories.ts

```diff
@@ -57,271 +66,60 @@ export const Default = {
-          }
-        ]
-      })}
-    </div>`;
-  }
-};
-
-/**
- *  This sample displays `sd-dialog` with two `sd-button` components in the footer slot. When incorporating multiple `sd-buttons`, it is recommended to use distinct button variants for clarity and consistency in user interactions.
- * 
- * You can use the CSS part `sd-dialog::part(footer)` to customize button layout based on your requirements. For instance, you may opt for columns instead of rows for mobile devices, and so on. To achieve that we used the additional style:
- * 
- * ```css
- * 
- *   @media (max-width: 414px) {
-          sd-dialog::part(footer){
-            flex-direction: column;
- *   }
- * }
-
- * ```
- * 
- * 
- */
-
-export const ExtendedFooter = {
-  name: 'Sample: Extended Footer',
-  parameters: {
-    controls: {
-      disable: true
-    }
-  },
-  render: () => {
-    return html`
-      <style>
-        @media (max-width: 414px) {
-          sd-dialog::part(footer) {
-            flex-direction: column;
-          }
-        }
-      </style>
-
-      <div style="height: 40vh;">
-        <sd-dialog open="" id="extended-footer"
-          ><p class="sd-paragraph">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
-            phasellus dui vel id. Velit in sed.
-          </p>
-          <h4 slot="headline" class="sd-headline sd-headline--size-3xl">Lorem Ipsum</h4>
-          <sd-button variant="secondary" slot="footer" class="w-full">Label</sd-button>
-          <sd-button slot="footer" class="w-full">Label</sd-button>
-        </sd-dialog>
-      </div>
-    `;
-  }
-};
-
-/**
- *  This sample displays `sd-dialog` with smaller headline sizes using the <span> tag instead of headings (eg: h2, h3, etc.).
- */
-
-export const SmallHeadline = {
-  name: 'Sample: Small Headline',
-  parameters: {
-    controls: {
-      disable: true
-    }
-  },
-  render: () => {
-    return html`
-      <div style="height: 40vh;">
-        <sd-dialog open="" id="small-headline"
-          ><p class="sd-paragraph">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
-            phasellus dui vel id. Velit in sed.
-          </p>
-          <span slot="headline" class="font-bold">Lorem Ipsum</span>
-          <sd-button slot="footer" class="w-full">Label</sd-button></sd-dialog
-        >
-      </div>
-    `;
-  }
+export const Headline = {
+  name: 'Headline',
+  render: () => html`
+    <div class="h-[40vh]">
+      <sd-dialog id="dialog" open headline="Headline">
+        <p class="sd-paragraph">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie phasellus
```

Comment:
Please adjust the text to match the Figma file
`Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam.`